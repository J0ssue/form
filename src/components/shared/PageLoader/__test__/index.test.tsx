import renderer from "react-test-renderer";
import PageLoader from "components/shared/PageLoader";

test("should show loading screen", () => {
  const tree = renderer.create(<PageLoader />).toJSON();
  expect(tree).toMatchSnapshot();
});
