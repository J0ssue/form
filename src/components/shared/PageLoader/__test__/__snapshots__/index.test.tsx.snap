// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show loading screen 1`] = `
<div>
  Loading...
</div>
`;
